src/routes/safe/components/Apps/communicator.ts

```diff
@@ -69,9 +69,12 @@ class AppCommunicator {
         }
       } catch (err) {
         this.send(err.message, msg.data.id, true)
-        // TODO: Allow passing method/message as an extra context
-        // Tweak CodedException class to accept it as a second argument
-        logError(Errors._901, `${msg.data.method} ${err.message}`)
+        logError(Errors._901, err.message, {
+          contexts: {
+            app: this.app,
```

Comment:
`safeApp: this.app` maybe?

Reply:
Since it's a 9** (errors reserved for safe apps) I doubt that makes any difference. No strong preference though

Reply:
For 900 you do use `safeApp` though.
Does `this.app` also contain a `url` property? Then they would be partially uniform.

Reply:
https://github.com/gnosis/safe-react/pull/2340/commits/680e49e41050abc5d21a68b7164187f160b087a5

```
export type SafeApp = {
  id: string
  url: string
  name: string
  iconUrl: string
  disabled?: boolean
  description: string
  error: boolean
  fetchStatus: SAFE_APP_FETCH_STATUS
}
```

yes it contains `url` prop